src/main/resources/changelog/num_1_users.xml

```diff
@@ -6,7 +6,7 @@
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet id="users_init" author="Oleg Kucherenko" context="structure">
-        <comment>Create roles and users tables</comment>
+        <comment>Create roles and users and country tables</comment>
```

Comment:
As we discussed earlier, we have to avoid dropping a database after changes. Create a separate changeSet in the num_2_workouts.xml, specify yourself as an author, move your changes to a new changeSet (note that you need alter users table by adding a new column).

Reference

https://docs.liquibase.com/change-types/add-column.html

(in this reference, omit catalogName and schemaName)